test/functional/ex_cmds/echo_spec.lua

```diff
@@ -11,6 +11,8 @@ local dedent = helpers.dedent
 local command = helpers.command
 local exc_exec = helpers.exc_exec
 local redir_exec = helpers.redir_exec
+local pcall_err = helpers.pcall_err
+local matches = helpers.matches
 
 describe(':echo', function()
```

Comment:
```suggestion
describe(':echo, :echomsg, :echoerr', function()
```

Then just put the new `it()` things in here.